presidio-analyzer/presidio_analyzer/predefined_recognizers/example_remote_recognizer.py

```diff
@@ -0,0 +1,97 @@
+import json
```

Comment:
Do we need this example both inside a file and in the documentation?
Can we link the documentation here so we won't need to maintain 2 places?

Reply:
I wasn't sure about it. I agree it should be in one place. Where in your opinion should it be? the code class is just sitting there with all the code but is not used.

Reply:
I am not sure what is best :) maybe we should put it up for discussion?

Reply:
@navalev WDYT?

Reply:
I would place it under docs/samples/python folder. would avoid building a package with sample code, and it can be referenced from the md files. This way we will maintain one point of code sample

Reply:
see my comment on the readme - IMO the class should be sample, and the __main__ should be in the readme

Reply:
Thanks @navalev . Good idea. I'll refactor. The main is tightly coupled with the sample, so I think I'll put it together with it.

Reply:
@navalev @shiranr fixed, please review. Also added a small section which was missing on adding predefined recognizers.